test/src/main/java/org/jvnet/hudson/test/PluginAutomaticTestBuilder.java

```diff
@@ -73,9 +78,27 @@ private static boolean notSkipTests(String propertyName) {
         return !Boolean.getBoolean("hpiTest.skip"+propertyName);
     }
 
-    public static class CliSanityTest extends HudsonTestCase {
```

Comment:
You can add as many test cases as you like so long as you use the same suite. For years there has been one test case; now I am adding a second. The few seconds it takes me to type that is insignificant compared to the ten or so seconds per suite per test run per plugin per user, adding up to countless days of computer time, some of it blocking developers from working.